snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/validation/SnomedQueryValidationRuleEvaluator.java

```diff
@@ -89,6 +72,16 @@
 		SnomedSearchRequestBuilder<?, PageableCollectionResource<SnomedComponent>> req = validationQuery
 				.prepareSearch();
 		
+		String extensionModules = params != null && params.containsKey(ValidationConfiguration.MODULES) ? params.get(ValidationConfiguration.MODULES).toString() : "";
```

Comment:
That toString() could cause an NPE, feel free to cast the value to String as is. E.g.:
```suggestion
		String extensionModules = params != null && params.containsKey(ValidationConfiguration.MODULES) ? Strings.nullToEmpty((String) params.get(ValidationConfiguration.MODULES)) : "";
```